gdk4-wayland/Cargo.toml

```diff
@@ -23,8 +23,8 @@ xkb_crate = ["xkb"]
 features = ["dox"]
 
 [dependencies]
-ffi = {path = "./sys", package = "gdk4-wayland-sys", version = "0.5.2"}
-gdk = {path = "../gdk4", package = "gdk4", version = "0.5.2"}
+ffi = {path = "./sys", package = "gdk4-wayland-sys", version = "0.5.3"}
+gdk = {path = "../gdk4", package = "gdk4", version = "0.5.3"}
```

Comment:
Is this needed?

Reply:
(same elsewhere)

Reply:
it is not actually but too lazy to fix it :P

Reply:
Let's not do that for 0.6 then :)